app/src/main/java/com/kickstarter/mock/services/MockApolloClient.kt

```diff
@@ -283,8 +283,8 @@ open class MockApolloClient : ApolloClientType {
         return Observable.just(1L)
     }
 
-    override fun sendVerificationEmail(): Observable<SendEmailVerificationMutation.Data> {
-        return Observable.just(
+    override fun sendVerificationEmail(): io.reactivex.Observable<SendEmailVerificationMutation.Data> {
+        return io.reactivex.Observable.just(
```

Comment:
As we update more classes, putting the library extensions will be unnecessary (there are a lot of classes using this, many still on rx1, which extends far beyond this ticket)